throttle/contract_tx_limiter_middleware.go

```diff
@@ -126,6 +127,25 @@ func NewContractTxLimiterMiddleware(cfg *ContractTxLimiterConfig,
 		if msgTx.VmType != vm.VMType_EVM {
 			return next(state, txBytes, isCheckTx)
 		}
+		if TxLimiter.inactiveDeployerContracts == nil || TxLimiter.contractDataLastUpdated+
+			cfg.ContractDataRefreshInterval < time.Now().Unix() {
+			ctx, err := createUserDeployerWhitelistCtx(state)
+			if err != nil {
+				return res, errors.Wrap(err, "throttle: context creation")
+			}
+			inactiveDeployerContracts, err := udw.GetInactiveDeployerContracts(ctx)
+			if err != nil {
+				return res, errors.Wrap(err, "throttle: inactiveDeployerContracts creation")
+			}
+			TxLimiter.inactiveDeployerContracts = inactiveDeployerContracts
+			// TxLimiter.contractDataLastUpdated will be updated after updating contractToTierMap
+		}
+		contractAddr := loom.UnmarshalAddressPB(msg.To)
+		// contracts which are deployed by deleted deployers should be throttled
+		if TxLimiter.inactiveDeployerContracts[contractAddr.String()] {
+			return res, errors.Wrap(err, "throttle: deployer of the contract is deleted")
```

Comment:
There's no error to wrap here.
-> `return res, errors.New("contract inactive")`

Reply:
done